getcoverage.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+CPP_ETHEREUM_PATH=$(pwd)
+
+if [ ! -d "$CPP_ETHEREUM_PATH/build/test" ]; then
```

Comment:
It works only if you have build in "build" dir. Instead of `CPP_ETHEREUM_PATH` point to build dir (e.g. `CPP_ETHEREUM_BUILD_DIR`)